src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -1,30 +1,30 @@
 package com.github.dockerjava.core.command;
 
-import static com.google.common.base.Preconditions.checkNotNull;
+import com.github.dockerjava.api.command.BuildImageCmd;
+import com.github.dockerjava.api.model.AuthConfigurations;
+import com.github.dockerjava.api.model.BuildResponseItem;
+import com.github.dockerjava.core.dockerfile.Dockerfile;
+import com.github.dockerjava.core.util.FilePathUtil;
 
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
 
-import com.github.dockerjava.api.command.BuildImageCmd;
-import com.github.dockerjava.api.model.AuthConfigurations;
-import com.github.dockerjava.api.model.BuildResponseItem;
-import com.github.dockerjava.core.dockerfile.Dockerfile;
-import com.github.dockerjava.core.util.FilePathUtil;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
- *
  * Build an image from Dockerfile.
- *
  */
 public class BuildImageCmdImpl extends AbstrAsyncDockerCmd<BuildImageCmd, BuildResponseItem> implements BuildImageCmd {
 
     private InputStream tarInputStream;
 
-    private String tag;
+    private Set<String> tags;
```

Comment:
This is private filed - to preserve backward compatibility I've left `getTag()` method.